debian/changelog

```diff
@@ -0,0 +1,5 @@
+truenas-pylibzfs (0.0.1-1) unstable; urgency=medium
```

Comment:
JFYI, I noticed that Debian packaging increments the revision number by 1, so we may likely see `0.0.1-2` being installed.